CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.1)
+
+# Install gtest which now lives on head. 
+cmake_policy(SET CMP0135 NEW)
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  # Specify the commit you depend on and update it regularly.
+  URL https://github.com/google/googletest/archive/5376968f6948923e2411081fd9372e71a59d8e77.zip
+  FIND_PACKAGE_ARGS NAMES GTest
+)
+FetchContent_MakeAvailable(googletest)
+
+# Setup CMAKE Module path for other conda packages
+SET(CMAKE_MODULE_PATH $ENV{$CONDA_PREFIX}/lib/cmake)
```

Comment:
Make sure find_package works with modules install in conda (CMake packages can be a pain tbh)